lib/Db/CoreRequestBuilder.php

```diff
@@ -52,15 +56,17 @@ class CoreRequestBuilder {
 	 * @param IL10N $l10n
 	 * @param IDBConnection $connection
 	 * @param ConfigService $configService
+	 * @param TimezoneService $timeZoneService
 	 * @param MiscService $miscService
 	 */
 	public function __construct(
 		IL10N $l10n, IDBConnection $connection, ConfigService $configService,
-		MiscService $miscService
```

Comment:
This change requires that constructor of class OCA\Circles\ShareByCircleProvider also be adjusted.